### DISCUSSION TITLE
Tero Subtitler 1.0.1.2

### DISCUSSION THREAD

**Discussion by URUWorks** [URUWorks/TeroSubtitler#190] [Announcements]
We've been hard at work making **Tero Subtitler** better for you. This includes bug fixes and performance enhancements. 
For more detailed information, check the [change log](https://raw.githubusercontent.com/URUWorks/TeroSubtitler/main/changelog.txt).

Note: If the Windows installer doesn’t work, it may be necessary to temporarily disable antivirus protection.

**Use it at your own risk**!

<hr /><em>This discussion was created from the release <a href='https://github.com/URUWorks/TeroSubtitler/releases/tag/1.0.1.2'>Tero Subtitler 1.0.1.2</a>.</em>

**Comment by nounours18200** [comment]
Hi,

How do we install it on Windows ???

When unzipped, we get a "TeroSubtitler-main" folder, but I see no ".exe" or ".msi" file inside ?

Thank you

**Reply by URUWorks** [reply]
I think you downloaded the wrong file, in the case of Windows, you should download: ts1012_win64_portable.zip or ts1012_win64_installer.exe

**Reply by nounours18200** [reply]
I understand, but where can I find these files ??

**Reply by nounours18200** [reply]
Well I have finally found them but not on GitHub. I have found them here: https://codecpack.co/download/tero-subtitler.html

Just a problem with the Faster Whisper: it is downloaded but not installed; maybe be because the portable file is on a NAS. I check this and will report the issue more clearly if it is confirmed.

Thank you very much ,

**Reply by nounours18200** [reply]
It looks like an truly excellent tool ! The only one that can compete with SubtitleEdit: same ergonomy, same efficiency !

I still have to check how it performs on generating video with burn-in subs.

The main issue relates to the audio waveform generator: it does not generate a waveform on my PC ??

**Reply by URUWorks** [reply]
in the case of Windows and faster-whisper please try this version: 

https://github.com/URUWorks/TeroSubtitler/issues/191

for waveform audio, you need to have ffmpeg.
